### ISSUE TITLE
Fall through crack between door and floor, die

### ISSUE THREAD

**Issue by btietz** [Interkarma/daggerfall-unity#2508]
I was in the tower of Balfiera, and while going clockwise up the stairs there's a doorway blocked by a sliding wall.  Triggered from elsewhere to open it up, obviously, but that's beside the point - if you walk against it, you fall through a minuscule crack, and die.  In the attached saved game, if you load, you'll be looking at a banner that covers the blocked doorway.  Simply go forward.  You will fall and die.  It is probably triggered by an undesirable gap in the model data which is obviously out of scope for DFU, but this seems like a likely bug in the "fall check" dimensions.  Like if there's a surface under the player within some distance in any direction, a fall shouldn't be possible, since you shouldn't actually fall through a crack.

[SAVE67.zip](https://github.com/Interkarma/daggerfall-unity/files/11194709/SAVE67.zip)

**Comment by btietz** [comment]
Never mind.  The sliding wall pushes you off the edge.  That was not immediately apparent initially.  Once I had disabled mods to verify what seemed like a bug, the reduced resolution made it even harder to catch.  It was when I put everything back, loaded that save, and tried to go around (not against the wall so as to not fall through the presumed crack) that I could actually see the wall coming at me.

**Comment by btietz** [comment]
should have closed with the preceding comment
